pkg/autoscaler/multiscaler.go

```diff
@@ -48,6 +48,8 @@ type DeciderSpec struct {
 	// The value of scaling metric per pod that we target to maintain.
 	// TargetValue <= TotalValue.
 	TargetValue float64
+	// The value of requests-per-second per pod that we target to maintain.
+	TargetRPS float64
```

Comment:
I don't think we need this. `TargetValue` is intended to be used for any metric.

Reply:
Ahh got it!